specification/storage/resource-manager/Microsoft.Storage/stable/2021-04-01/storage.json

```diff
@@ -0,0 +1,4549 @@
+          ],
+          "x-ms-enum": {
+            "name": "schedule",
+            "modelAsString": true
+          }
+        },
+        "objectType": {
+          "type": "string",
+          "description": "This is a required field. This field specifies the scope of the inventory created either at the blob or container level.",
+          "enum": [
+            "Blob",
+            "Container"
+          ],
+          "x-ms-enum": {
+            "name": "objectType",
+            "modelAsString": true
+          }
+        },
+        "schemaFields": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "This is a required field. This field specifies the fields and properties of the object to be included in the inventory. The Schema field value 'Name' is always required. The valid values for this field for the 'Blob' definition.objectType include 'Name, Creation-Time, Last-Modified, Content-Length, Content-MD5, BlobType, AccessTier, AccessTierChangeTime, Expiry-Time, hdi_isfolder, Owner, Group, Permissions, Acl, Snapshot, VersionId, IsCurrentVersion, Metadata, LastAccessedTime'. The valid values for 'Container' definition.objectType include 'Name, Last-Modified, Metadata, LeaseStatus, LeaseState, LeaseDuration, PublicAccess, HasImmutabilityPolicy, HasLegalHold'. Schema field values 'Expiry-Time, hdi_isfolder, Owner, Group, Permissions, Acl' are valid only for Hns enabled accounts."
+        }
+      },
+      "required": [
+        "format",
+        "schedule",
+        "objectType",
+        "schemaFields"
+      ],
+      "description": "An object that defines the blob inventory rule."
+    },
+    "BlobInventoryPolicyFilter": {
+      "properties": {
+        "prefixMatch": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "An array of strings for blob prefixes to be matched."
+        },
+        "blobTypes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "An array of predefined enum values. Valid values include blockBlob, appendBlob, pageBlob. Hns accounts does not support pageBlobs. This field is required when definition.objectType property is set to 'Blob'."
+        },
+        "includeBlobVersions": {
+          "type": "boolean",
+          "description": "Includes blob versions in blob inventory when value is set to true. The definition.schemaFields values 'VersionId and IsCurrentVersion' are required if this property is set to true, else they must be excluded."
+        },
+        "includeSnapshots": {
+          "type": "boolean",
+          "description": "Includes blob snapshots in blob inventory when value is set to true. The definition.schemaFields value 'Snapshot' is required if this property is set to true, else it must be excluded."
+        }
+      },
+      "required": [
+        "blobTypes"
```

Comment:
@HimanshuChhabra 
Per Ruchi Bansal, "In filter definition, we have 4 properties (prefixMatch, blobTypes, includeSnapshots and includeBlobVersions) where prefixMatch is valid for both Blob and Container object types and other three are valid only for Blob inventory. "

So for container inventory, we should can have filter with only prefix. In this way, the "blobTypes" should not be required, or we can create a valid filter for container inventory.